### DISCUSSION TITLE
Implement feature highlight

### DISCUSSION THREAD

**Discussion by oottoohh** [wix/react-native-ui-lib#2676] [Q&A]
any one can help me guys, i still struggling when implement this FeatureHighlight to my app project. so i need highlight component inside header, my header is standalone component. i put my header inside navigator. so i can't use ref local state to identify for feature highlight. i have been try set ref to mobx but still fail any one have suggestion to implement this ?
another way, i try using props highlightframe so the feature can be highlight based on position of component but the problem is error show up infinitely i still confuse why this happened. the error is :

`Invariant Violation: [36595,"RCTView",511,{"onLayout":true,"collapsable":false,"accessibilityState":{},"accessibilityValue":{},"position":"absolute","padding":32,"marginRight":45,"alignItems":"flex-start","opacity":0,"top":"<<NaN>>","pointerEvents":"box-none"}] is not usable as a native method argument`

**Comment by thisisadithk** [comment]
@oottoohh This error comes with when you tie the addTarget function to a View. Pls try adding it to a TouchableOpacity and it should not give you this error. Not sure what the fix is for the issue but seems to be a bug already raised.. 
https://github.com/wix/react-native-ui-lib/issues/2653
